react/components/molecules/key-values/key-values.js

```diff
@@ -78,7 +78,7 @@ export class KeyValues extends PureComponent {
     };
 
     _shouldShow = item => {
-        return Boolean(item.value) || this.props.showUnset;
+        return Boolean(item.value || this.props.showUnset) && !item.section;

```

Comment:
is this change still needed?

Reply:
yes, it's either that or change the logic of the template, I think this is the best option